posthog/hogql/database/test/__snapshots__/test_database.ambr

```diff
@@ -1,6 +1,18 @@
 # serializer version: 1
 # name: TestDatabase.test_database_warehouse_joins_persons_poe_v2
-  'SELECT events__some_field.key AS key FROM events LEFT JOIN (SELECT groups.key AS key, key AS events__some_field___key FROM (SELECT groups.group_type_index AS index, groups.group_key AS key FROM groups WHERE equals(groups.team_id, 218) GROUP BY groups.group_type_index, groups.group_key) AS groups) AS events__some_field ON equals(replaceRegexpAll(nullIf(nullIf(JSONExtractRaw(events.person_properties, %(hogql_val_0)s), \'\'), \'null\'), \'^"|"$\', \'\'), events__some_field.events__some_field___key) WHERE equals(events.team_id, 218) LIMIT 10000'
+  '''
+  
+  SELECT events__some_field.key AS key 
+  FROM events LEFT JOIN (
+  SELECT groups.key AS key, key AS events__some_field___key 
+  FROM (
+  SELECT groups.group_type_index AS index, groups.group_key AS key 
+  FROM groups 
+  WHERE equals(groups.team_id, 420) 
+  GROUP BY groups.group_type_index, groups.group_key) AS groups) AS events__some_field ON equals(replaceRegexpAll(nullIf(nullIf(JSONExtractRaw(events.person_properties, %(hogql_val_0)s), ''), 'null'), '^"|"$', ''), events__some_field.events__some_field___key) 
+  WHERE equals(events.team_id, 420) 
```

Comment:
This still has the team id in the snapshot - I think we want that replaced with another character to stop it flopping about, no?

Reply:
team_id is always replaced to that number now